server/utils/helpers/activityTimeSlotMappers.ts

```diff
@@ -95,6 +96,62 @@ export function slotsToCustomTimeSlots(scheduleWeek: number, slots: Slot[]): Wee
   return customSlots
 }
 
+export function regimeSlotsToSchedule(
```

Comment:
I think this might be pretty similar to another function I built the other day for Dave's page, though now I realise I didn't put it in the most helpful place (in this file) - getApplicableDaysAndSlotsInRegime
Happy for this to go in though and we can sort it out afterwards?